frontend/packages/console-app/src/components/volume-snapshot/create-volume-snapshot/create-volume-snapshot.tsx

```diff
@@ -201,6 +201,11 @@ const CreateSnapshotForm = withHandlePromise<SnapshotResourceProps>((props) => {
       metadata: {
         name: snapshotName,
         namespace: getNamespace(pvcObj),
+        annotations: {
+          'snapshot.storage.kubernetes.io/pvc-access-modes': pvcObj.spec.accessModes.join(','),
+          'snapshot.storage.kubernetes.io/pvc-storage-class': pvcObj.spec.storageClassName,
+          'snapshot.storage.kubernetes.io/pvc-volume-mode': pvcObj.spec.volumeMode,
```

Comment:
Define these annotations as a const and use in both the files.

Reply:
done